subgraphs/makerdao/src/utils/numbers.ts

```diff
@@ -0,0 +1,43 @@
+import { BigDecimal, BigInt, Bytes, Address } from "@graphprotocol/graph-ts";
+import {
+  BIGINT_NEG_ONE,
+  BIGDECIMAL_NEG_ONE,
+  BIGDECIMAL_ZERO,
+  BIGINT_ZERO,
+  BIGDECIMAL_TWO,
+  BIGDECIMAL_ONE,
+  BIGDECIMAL_SIX,
+  BIGDECIMAL_TWELVE,
+  BIGDECIMAL_THREE,
+  BIGDECIMAL_ONE_THOUSAND,
+} from "../common/constants";
+
+export function bigIntToBDUseDecimals(quantity: BigInt, decimals: i32 = 18): BigDecimal {
```

Comment:
This is nice. Can you add this to the common lib in the reference subgraph?

Reply:
Will do after I get the subgraph to work correctly. Some of the functions I borrowed from @[tannishmango](https://github.com/tannishmango)'s implement

Reply:
Looks like this function is already in there `_reference/src/common/utils/numbers.ts`: `bigIntToBigDecimal`